### PR TITLE
Add 2022 English CFP

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: 2022/cfp.md [hunk 1]
```diff
@@ -1,7 +1,168 @@
-# Call for Proposals
+# JuliaCon 2022 CFP
+*This document is available in French, German, Spanish, Russian, Simplified Chinese, ... (Missing your own? [Submit a PR](https://github.com/JuliaCon/www.juliacon.org/pulls))
```
[the remark this answers is not in the document]
Comment (kshyatt) [reply 2022/cfp.md:2]: Let's wait until we get a few merged, in order to not break links.

File: 2022/cfp.md [hunk 2]
```diff
@@ -1,7 +1,168 @@
...
+
+If you are submitting a workshop or birds-of-a-feather session that you think would particularly benefit from being held at a certain time, pleaset note this in your proposal. For example, if you would like to present a workshop in Mandarin, and want it to be held at a time convenient for attendees in Asia, indicate this in your abstract.
```
Comment (bicycle1885) [inline 2022/cfp.md:59]: pleaset -> please?

File: 2022/cfp.md [hunk 3]
```diff
@@ -1,7 +1,168 @@
...
+- Julia’s compiler, tooling, and ecosystem
+- Numerical and mathematical optimization
+- Scientific computing
+- Software engineering best practices
+- Statistics, machine learning, and AI
+- Computational humanities and social science
+
+The most important consideration is whether the topic would be interesting to the Julia community.
+
+# Proposal Types
```
Comment (gdalle) [inline 2022/cfp.md:26]:
```suggestion
## Proposal Types
```

File: 2022/cfp.md [hunk 4]
```diff
@@ -1,7 +1,168 @@
...
+
+### Experiences
+The purpose of this track is to welcome new members in the community. Example topics include your favorite features in Julia, how Julia has helped you solve a certain problem, an interesting problem you solved with Julia in a university class, or, more generally, how Julia has been a good addition to your workflow. These presentations will each last three minutes and will be interspersed throughout different sessions in the conference. We encourage you to show contact information such as twitter handle or github handle or email on your slides if you’d welcome any offline questions, as there won’t be allocated time for questions in this track. We especially welcome newcomers to Julia to share their impressions with the community using this track. If you are a teacher or professor using Julia in your teaching, we encourage you to invite your students to apply to share their experiences with the Julia community.
+
+-----
+
+When submitting your proposal (abstract), you may select multiple categories for consideration.
+
+# Submission details
```
Comment (gdalle) [inline 2022/cfp.md:55]:
```suggestion
## Submission details
```

File: 2022/cfp.md [hunk 5]
```diff
@@ -1,7 +1,168 @@
...
+
+### How to contact us
+
+You can reach us with questions and concerns at juliacon@julialang.org.
+
+# Conference Code of Conduct
+
+JuliaCon is dedicated to providing a positive conference experience for all attendees, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or national and ethnic origin. We encourage respectful and considerate interactions between attendees and do not tolerate harassment of conference participants in any form. For example, offensive or sexual language and imagery is not appropriate for any conference venue, including formal talks and networking between sessions. Conference participants violating these standards may be sanctioned or expelled from the conference (without a refund) at the discretion of the conference organizers. Our anti-harassment policy can be found [here](https://juliacon.org/2022/coc).
+
+# Appendix
```
Comment (gdalle) [inline 2022/cfp.md:86]:
```suggestion
## Appendix
```

File: 2022/cfp.md [hunk 6]
```diff
@@ -1,7 +1,168 @@
...
+
+## Review Guidelines and Process
+
+For reference, below are the guidelines and processes that readers will use in reviewing your submission.
+
+The role of reviewers is to ensure the quality of the content presented at JuliaCon.
+
+## Conflict of interest
```
Comment (gdalle) [inline 2022/cfp.md:94]:
```suggestion
### Conflict of interest
```

File: 2022/cfp.md [hunk 7]
```diff
@@ -1,7 +1,168 @@
...
+
+In any case of conflict of interest, the reviewer commits to withdraw from a review and signal it to the organizers to find a replacement quickly. No reviewer should enter a review on any talk in which they are an author or have another form of conflict of interest. See the [PNAS guidelines](https://www.pnas.org/page/authors/conflict-of-interest) for a definition and examples. Conflicts of interest include any work or authors with which the reviewer has "any association that poses or could be perceived as a financial or intellectual conflict of interest" (PNAS guidelines above).
+
+## Code of Conduct
```
Comment (gdalle) [inline 2022/cfp.md:98]:
```suggestion
### Code of Conduct
```

File: 2022/cfp.md [hunk 8]
```diff
@@ -1,7 +1,168 @@
...
+
+The reviewer commits to reading and respecting the conference [Code of Conduct](https://juliacon.org/2021/coc) in the assessment and all communications during the review process.
+
+If a submitted abstract does not comply with the Code of Conduct, the reviewer should refer it to the organizing committee.
+
+## Criteria for the reviews
```
Comment (gdalle) [inline 2022/cfp.md:104]:
```suggestion
### Criteria for the reviews
```

File: 2022/cfp.md [hunk 9]
```diff
@@ -1,7 +1,168 @@
...
+- Scientific computing
+- Software engineering best practices
+- Statistics, machine learning, and AI
+- Computational humanities and social science
+
+4. If the format requested by the author (experience, lightning talk, talk, workshop, poster, minisymposium, birds-of-a-feather) does not seem appropriate, the reviewer can signal it and suggest another one.
+
+5. Use cases of Julia in an enterprise environment are in general of interest to the conference. In particular, feedback on product development using or interacting with Julia and its ecosystem are welcome. However, talks and posters are not a suitable venue for product placement.
+
+## Scoring Criteria
```
Comment (gdalle) [inline 2022/cfp.md:130]:
```suggestion
### Scoring Criteria
```

File: 2022/cfp.md [hunk 10]
```diff
@@ -1,7 +1,168 @@
...
+
+4. Significance to the community. Is this something that will change the way a lot of other people use Julia or its package ecosystem? Higher scoring proposals should be more significant to Julia users. Note that this does not require scientific significance, just significance as a software or tutorial to users of Julia.
+
+5. Topic diversity. As a community we value the diversity of applications. Proposals which are targeting new areas and fields for the Julia community to expand should be given some credit.
+
+6. Soundness. Proposals should be technically sound. Glaring incorrectness should be highlighted and taken into account.
+
+7. Classification. The criteria will be stricter for longer presentations.
+
+## Review Process
```
Comment (gdalle) [inline 2022/cfp.md:148]:
```suggestion
### Review Process
```

File: 2022/cfp.md [hunk 11]
```diff
@@ -1,7 +1,168 @@
...
+
+- After the two review stages is the selection stage. The top proposals will be accepted and the bottom proposals will be rejected or re-classified and counted as a middle proposal. All proposals in the middle will then be marked for extra consideration. The amount of "top proposals" to be considered for stage 1 selection is dependent on the number of available speaker slots vs the number of proposals, and generally is reserved for average scores of 4+ without any comments of concern.
+
+- In the second selection stage, the review committee will take the reviewer scores, comments, and re-classifications to decide the final acceptances. Rejected proposals will be discussed for re-classification before a full rejection.
+
+- After all proposals have been classified, acceptances and rejections will be released.
+
+- A special symposium for Google Summer of Code student lightning talks will be reserved. This is kept for later in the process since the approval/rejection of proposals is BEFORE GSoC students are accepted. These proposals will be directly reviewed by the review committee.
+
+## Review Comments
```
Comment (gdalle) [inline 2022/cfp.md:162]:
```suggestion
### Review Comments
```

**Pull request merged**